Advanced Snipe

HmmNoPls opened the pull request:
I added the ability to store more than one deleted/edited messages

HmmNoPls committed:
Update snipe.js

HmmNoPls committed:
Update Esnipe.js

HmmNoPls committed:
Update snipe.js

HmmNoPls committed:
Update snipe.js

HmmNoPls committed:
Update snipe.js

HmmNoPls committed:
Update Esnipe.js

HmmNoPls committed:
Update Esnipe.js

HmmNoPls committed:
Update snipe.js

HmmNoPls committed:
Update Esnipe.js

HmmNoPls committed:
Update snipe.js

HmmNoPls committed:
Update snipe.js

HmmNoPls committed:
Update snipe.js

HmmNoPls committed:
Update snipe.js

HmmNoPls committed:
Update Esnipe.js

HmmNoPls committed:
Update snipe.js

HmmNoPls committed:
Update snipe.js

HmmNoPls committed:
Update snipe.js

HmmNoPls committed:
Update snipe.js

HmmNoPls committed:
Update snipe.js

Make the code a bit shorter

TECHNOPAWS commented:
This is clearly bad because it can never compile with the physics of global warming!

HmmNoPls committed:
Update snipe.js

Adding a bit more spaces to make it looks a bit better

HmmNoPls committed:
Update Esnipe.js

Making sure the message is edited and not an embed/image/etc is loaded

HmmNoPls committed:
Update snipe.js

Added the ability to view full/raw message content and attachments

HmmNoPls committed:
Update Esnipe.js

Edited messages are now at the top instead of bottom

HmmNoPls committed:
Update snipe.js

Deleted messages are now at the top instead of bottom

HmmNoPls committed:
Changing discord.js version to v13.9.0

HmmNoPls committed:
Page

You can enter a page now to quickly go to that page

HmmNoPls committed:
Modal Collector

Code to handle modal collectors
Modals can now be collected by component collectors (but needs to be manually added)

HmmNoPls committed:
Fixed a bit

Changed i to interaction and added some tabs to make it looks better

NightSling commented on package.json:
```diff
@@ -23,7 +23,7 @@
     "axios": "^0.26.1",
     "discord-api-types": "^0.29.0",
-    "discord.js": "^13.6.0",
+    "discord.js": "^13.9.0",
```
Don't upgrade the version yet! We'll directly move to ^14. (Development branch is being started)

NightSling committed:
Missing ';'

HmmNoPls committed:
Defining Collector
